ignite/templates/app/files/app/app_config.go.plush

```diff
@@ -213,75 +227,85 @@ var (
+					Bech32PrefixConsensus: AccountAddressPrefix + "valcons",
 				}),
 			},
 			{
-				Name:   feegrant.ModuleName,
-				Config: appconfig.WrapAny(&feegrantmodulev1.Module{}),
-			},
-			{
-				Name:   govtypes.ModuleName,
-				Config: appconfig.WrapAny(&govmodulev1.Module{}),
-			},
-			{
-				Name:   crisistypes.ModuleName,
-				Config: appconfig.WrapAny(&crisismodulev1.Module{}),
-			},
-			{
-				Name:   consensustypes.ModuleName,
-				Config: appconfig.WrapAny(&consensusmodulev1.Module{}),
-			},
+                Name:   slashingtypes.ModuleName,
+                Config: appconfig.WrapAny(&slashingmodulev1.Module{}),
+            },
+            {
+                Name:   paramstypes.ModuleName,
+                Config: appconfig.WrapAny(&paramsmodulev1.Module{}),
+            },
+            {
+                Name:   "tx",
+                Config: appconfig.WrapAny(&txconfigv1.Config{}),
+            },
+            {
+                Name:   genutiltypes.ModuleName,
+                Config: appconfig.WrapAny(&genutilmodulev1.Module{}),
+            },
+            {
+                Name:   authz.ModuleName,
+                Config: appconfig.WrapAny(&authzmodulev1.Module{}),
+            },
+            {
+                Name:   upgradetypes.ModuleName,
+                Config: appconfig.WrapAny(&upgrademodulev1.Module{}),
+            },
+            {
+                Name:   distrtypes.ModuleName,
+                Config: appconfig.WrapAny(&distrmodulev1.Module{}),
+            },
+            {
+                Name:   evidencetypes.ModuleName,
+                Config: appconfig.WrapAny(&evidencemodulev1.Module{}),
+            },
+            {
+                Name:   minttypes.ModuleName,
+                Config: appconfig.WrapAny(&mintmodulev1.Module{}),
+            },
+            {
+                Name: group.ModuleName,
+                Config: appconfig.WrapAny(&groupmodulev1.Module{
+                    MaxExecutionPeriod: durationpb.New(time.Second * 1209600),
+                    MaxMetadataLen:     255,
+                }),
+            },
+            {
+                Name:   feegrant.ModuleName,
+                Config: appconfig.WrapAny(&feegrantmodulev1.Module{}),
+            },
+            {
+                Name:   govtypes.ModuleName,
+                Config: appconfig.WrapAny(&govmodulev1.Module{}),
+            },
+            {
+                Name:   crisistypes.ModuleName,
+                Config: appconfig.WrapAny(&crisismodulev1.Module{}),
+            },
+            {
+                Name:   consensustypes.ModuleName,
+                Config: appconfig.WrapAny(&consensusmodulev1.Module{}),
+            },
+            {
+                Name:   circuittypes.ModuleName,
+                Config: appconfig.WrapAny(&circuitmodulev1.Module{}),
+            },
 			// this line is used by starport scaffolding # stargate/app/moduleConfig
 		},
-	})
+	}),
+		depinject.Supply(
```

Comment:
I would put this in the app.go instead of here as it makes it a bit more readable as well. We should do that in simapp in the sdk as well.

Reply:
put it into the app.go break the chain

```
✘ Panic: interface conversion: module.AppModuleBasic is module.coreAppModuleBasicAdaptor, not genutil.AppModuleBasic                                                    
```

Reply:
It shouldn't: https://github.com/cosmosregistry/chain-minimal/blob/main/app/app.go#L86-L96

Reply:
great, I got it, moved